cmd/argo/commands/root.go

```diff
@@ -62,10 +64,16 @@ If you're using the Argo Server (e.g. because you need large workflow support or
 
 	// global log level
 	var logLevel string
+	var verbose bool
 	command.PersistentPreRun = func(cmd *cobra.Command, args []string) {
+		if verbose {
+			logLevel = "debug"
+		}
```

Comment:
Perhaps make the field a string field? What if we want to add more fine-tuned logging in the future?